mkdocs-base.yml

```diff
@@ -195,6 +195,7 @@ nav:
     - Configuration:
           - "Application and system users": users.md
           - "Changing MongoDB options": options.md
+          - options-pbm.md
```

Comment:
do we need a title?

Reply:
Not necessarily, mkdocs renders the title from Heading 1 of the page